extension/js/common/platform/xss.ts

```diff
@@ -131,7 +131,7 @@ export class Xss {
           img.remove(); // just skip images
         } else if (!src) {
           img.remove(); // src that exists but is null is suspicious
-        } else if (imgHandling === 'IMG-TO-LINK') {
+        } else if (imgHandling === 'IMG-TO-LINK' && !img.classList.contains('cid_to_base64_img')) {
```

Comment:
I think we should have consistent behavior for inline images. As currently images with cid will be shown immediately on load, while other inline images will need to be clicked before showing (with `IMG-TO-LINK` image handling)

Reply:
I was also considering that. Should we display the 'show inline' button and then show the image when the user clicks on it? Or should we simply show the inline image immediately? I don't think there are any security issues with showing the inline image right away, though.

Reply:
Yes, I think showing inline images immediately without additional button will be more convenient

Reply:
But then do we need `IMG-TO-LINK` type?
Do we have to remove this logic and update UI test too?

https://github.com/FlowCrypt/flowcrypt-browser/blob/6ff785cc55d3f892ffdea8609871628f8cfb2884/extension/js/common/platform/xss.ts#L9

Reply:
Seems like `IMG-TO-LINK` isn't needed after this change.

And we're already showing remote images immediately after load, so this update will make all images to behave the same - inline and remote.

Reply:
Yeah, I also agree

Reply:
But one thing I'm unsure is that `show image` button was customer request before then.

Reply:
Maybe @tomholub, I think you can confirm why we displayed `show image` button before.

Reply:
Yeah, I was thinking about it, but probably it was for security reasons (as for remote images).
But as we allow to load remote images now, then it won't be an issue to have inline images to be auto-showed too

Reply:
Makes sense